idaes/models_extra/power_generation/properties/tests/test_sofc_surrogates.py

```diff
@@ -0,0 +1,125 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+
+"""
+Tests for the SOFC keras surrogates.
+
+Author: A. Noring
+"""
+
+import pytest
+
```

Comment:
Since Tensorflow is an optional dependency for IDAES, this test file should be skipped if `tensorflow` is not available, e.g. by adding `pytest.importorskip("tensorflow")` anywhere before importing `idaes.models_extra.power_generation.properties.sofc.sofc_keras_surrogate`.